bootstrap.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+sudo yum -y groupinstall 'Development Tools'
+sudo yum -y install java-1.8.0-openjdk
+
```

Comment:
@olivermel We need to install clojure and lein as well, somehow this ended up getting dropped from the file, see instructions here: http://www.freshblurbs.com/blog/2011/01/29/installing-and-configuring-clojure-centos-5.html